crates/re_build_web_viewer/src/lib.rs

```diff
@@ -108,11 +97,8 @@ pub fn build(
         "--target-dir",
         target_wasm_dir.as_str(),
         "--no-default-features",
+        "--features=analytics",
```

Comment:
unrelated change?

Reply:
no, if you look closely this moved in from the if/else further down - both branches enabled the analytics feature